azerothcore/azerothcore-wotlk
# fix(Scripts/UBRS): Drakkisath's conflagrate should drop threat tempor…

…arily

also adjust it's timer slightly



## Changes Proposed:
-  Reset threat temporarily for Conflag. targets
-  Adjust timer according to feedback

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9003

## SOURCE:


## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. engage drakkisath with 2 characters
2. notice threat drops when he casts conflag and he goes to the next target
3. notice that threat is readded once conflag expires (according to feedback it should be temporary)

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
